### PR TITLE
Change release to multi task

### PR DESCRIPTION
**Pull request by zwhitchcox**
More concise and consistent with the other multitasks

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -137,12 +137,7 @@ module.exports = function(grunt) {
   ]);
 
   // Automate the release!
-  var releaseTasks = require('./grunt/tasks/release');
-  grunt.registerTask('release:setup', releaseTasks.setup);
-  grunt.registerTask('release:bower', releaseTasks.bower);
-  grunt.registerTask('release:docs', releaseTasks.docs);
-  grunt.registerTask('release:msg', releaseTasks.msg);
-  grunt.registerTask('release:starter', releaseTasks.starter);
+  grunt.registerMultiTask('release' require('./grunt/tasks/release'));
```
Comment (yangshun) [inline Gruntfile.js]: missing a comma?

**Comment by zwhitchcox** [comment]
Oops...not really a way you can test this, unfortunately. Thank goodness for Travis :)

**Comment by zpao** [comment]
Thanks. Can you squash to a single commit? Then I'll merge

**Comment by zwhitchcox** [comment]
All right, dunzo!

**Pull request merged**